lib/model/QubitTerm.php

```diff
@@ -336,7 +337,24 @@ public function delete($connection = null)
 
     QubitSearch::getInstance()->delete($this);
 
-    parent::delete($connection);
+    // Replicate parent class delete functionality, skipping nested set updating
```

Comment:
I agree with @MikeFE.

@mcantelon have you investigated how hard would it be to do this for all models with nested set in their base class, by extending and modifying sfPropelBuildModelTask and related classes. I'd guess it's not an easy change, but I think it will be safer than this.